Source/Core/Core/HW/HSP/HSP.cpp

```diff
@@ -0,0 +1,67 @@
+// Copyright 2018 Dolphin Emulator Project
+// Licensed under GPLv2+
+// Refer to the license.txt file included.
```

Comment:
These need to be updated to the new format:

```C++
// Copyright 2018 Dolphin Emulator Project
// SPDX-License-Identifier: GPL-2.0-or-later
```

(you could also bump the year to 2022, though I don't have a strong preference to that)